astroplan/plots/time_dependent.py

```diff
@@ -65,9 +71,15 @@ def plot_airmass(target, observer, time, ax=None, style_kwargs=None):
 
     TODO:
         1) Timezones?
-        2) Dark plot option.
     """
+    # Import matplotlib, set style sheet
```

Comment:
These ~ 4 lines ob boilerplate code at the beginning of each plot function could be moved to a utility function `_set_mpl_style(style)`, no?